confluence/server/scio_search/src/main/java/com/askscio/atlassian_plugins/confluence/impl/ScioSearchServletFilter.java

```diff
@@ -69,7 +69,11 @@ public void doFilter(
     }
 
     final HttpServletRequest httpreq = (HttpServletRequest) servletRequest;
-    if (!httpreq.getRequestURI().contains("viewpage")
+    // Saving a page or blogpost fires: PUT http://confluence-server:8090/rest/api/content/65603?status=draft
+    if (httpreq.getRequestURI().startsWith("/rest/api/content/") &&
+        ("PUT".equals(httpreq.getMethod()) || "POST".equals(httpreq.getMethod()) || "DELETE".equals(httpreq.getMethod()))) {
+      logger.debug("Save url: " + httpreq.getRequestURI());
```

Comment:
nit: can you add the method also in the log

Reply:
done.